lib/data_fifo/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+zephyr_library_sources(
+	data_fifo.c
+)
+
+zephyr_include_directories(.)
```

Comment:
There are no .h files here?

Reply:
Correct. NCS includes *.h files from a different location.

Reply:
But then "zephyr_include_directories(.)" is not needed?